indico/modules/events/reminders/templates/reminders.html

```diff
@@ -43,9 +51,12 @@
             <div class="i-box-header">
                 <div class="i-box-title">{%- trans %}Pending Reminders{% endtrans -%}</div>
                 <div class="i-box-buttons">
-                    <a class="i-button icon-plus" href="{{ url_for('.add', event) }}">
-                        {%- trans %}Add Reminder{% endtrans -%}
-                    </a>
+                    <a class="i-button icon-plus" data-href="{{ url_for('.add', event) }}"
+                       data-ajax-dialog
+                       data-hide-page-header
```

Comment:
use `jsonify_template()` instead of WP....render_template()` instead and remove the extends and blocks in `edit_reminder.html` - then you don't need to specify this at all